drivers/wifi/siwx917/siwx917_wifi_socket.c

```diff
@@ -0,0 +1,422 @@
+/*
+ * Copyright (c) 2024 Silicon Laboratories Inc.
+ *
+ * The licensor of this software is Silicon Laboratories Inc. Your use of this
```

Comment:
Is there a reason to not use Apache-2.0 for this?

Reply:
Bad habits. I will fix that tomorrow.